From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: John Mulligan <jmulligan@redhat.com>
Date: Wed, 16 Apr 2025 14:55:08 -0400
Subject: [PATCH] python-common/cryptotools: give the parsers more sensible
 names

Name the parser objects after their functions and not `foo` and `bar`.

Signed-off-by: John Mulligan <jmulligan@redhat.com>
Signed-off-by: Thomas Lamprecht <t.lamprecht@proxmox.com>
---
 .../ceph/cryptotools/cryptotools.py           | 26 +++++++++----------
 1 file changed, 12 insertions(+), 14 deletions(-)

diff --git a/src/python-common/ceph/cryptotools/cryptotools.py b/src/python-common/ceph/cryptotools/cryptotools.py
index 9d2f6d6db04..0b2dc828b79 100644
--- a/src/python-common/ceph/cryptotools/cryptotools.py
+++ b/src/python-common/ceph/cryptotools/cryptotools.py
@@ -171,28 +171,26 @@ if __name__ == "__main__":
     subparsers = parser.add_subparsers(required=True)
 
     # create the parser for the "password_hash" command
-    parser_foo = subparsers.add_parser('password_hash')
-    parser_foo.set_defaults(func=password_hash)
+    parser_password_hash = subparsers.add_parser('password_hash')
+    parser_password_hash.set_defaults(func=password_hash)
 
     # create the parser for the "create_self_signed_cert" command
-    parser_bar = subparsers.add_parser('create_self_signed_cert')
-    parser_bar.add_argument('--private_key', required=False, action='store_true')
-    parser_bar.add_argument('--certificate', required=False, action='store_true')
-    parser_bar.set_defaults(func=create_self_signed_cert)
+    parser_cssc = subparsers.add_parser('create_self_signed_cert')
+    parser_cssc.add_argument('--private_key', required=False, action='store_true')
+    parser_cssc.add_argument('--certificate', required=False, action='store_true')
+    parser_cssc.set_defaults(func=create_self_signed_cert)
 
     # create the parser for the "certificate_days_to_expire" command
-    parser_bar = subparsers.add_parser('certificate_days_to_expire')
-    parser_bar.set_defaults(func=certificate_days_to_expire)
+    parser_dte = subparsers.add_parser('certificate_days_to_expire')
+    parser_dte.set_defaults(func=certificate_days_to_expire)
 
     # create the parser for the "get_cert_issuer_info" command
-    parser_bar = subparsers.add_parser('get_cert_issuer_info')
-    parser_bar.add_argument('--org_name', required=False, action='store_true')
-    parser_bar.add_argument('--cn', required=False, action='store_true')
-    parser_bar.set_defaults(func=get_cert_issuer_info)
+    parser_gcii = subparsers.add_parser('get_cert_issuer_info')
+    parser_gcii.set_defaults(func=get_cert_issuer_info)
 
     # create the parser for the "verify_tls" command
-    parser_bar = subparsers.add_parser('verify_tls')
-    parser_bar.set_defaults(func=verify_tls)
+    parser_verify_tls = subparsers.add_parser('verify_tls')
+    parser_verify_tls.set_defaults(func=verify_tls)
 
     # parse the args and call whatever function was selected
     args = parser.parse_args()
